Gems/ScriptCanvas/Code/Include/ScriptCanvas/Debugger/Logger.h

```diff
@@ -79,9 +79,9 @@ namespace ScriptCanvas
             bool m_logExecutionOverride = false;
 
 #if defined(SC_EXECUTION_TRACE_ENABLED)
-            ExecutionLogAsset m_logAsset;
+            ExecutionLogData m_logData;
 #endif
-            ScriptCanvas::Debugger::Target m_target;
+            ScriptCanvas::Debugger::Target* m_target;
```

Comment:
not for now but might be wise to have a weakptr there in  the future

Reply:
I'll keep it in mind, don't have time to dive into it more this weekend